tox.ini

```diff
@@ -3,21 +3,15 @@ envlist = {py27,py34,py35,py36}-flake8{2,3},pypy,pep8,py3pep8,setuppy,manifest
 
 [testenv]
 deps =
-    coverage==3.7
     pytest
+    pytest-cov
     flake82: flake8<3.0
     flake83: flake8>3.0
     pylama
     pycodestyle>=2.0
     pyflakes>=1.3.0
 commands =
-    coverage run --source=flake8_import_order/,tests/ -m pytest --capture=no --strict {posargs}
-    coverage report -m
-
-# Temporarily disable coverage on pypy because of performance problems with
```

Comment:
On my box, the tests on pypy take 4.4 seconds with coverage to 1.2 seconds without. I'm assuming that's not significant enough to constitute the kind of problem that would make you disable coverage there.